arches/app/models/card.py

```diff
@@ -202,6 +204,14 @@ def save(self):
 
         """
         with transaction.atomic():
+            if self.graph.publication is not None:
+                ret = {
+                    "title": _("Unable to Save Card"),
+                    "message": _(
+                        "You cannot edit a card in a published graph.  Please unpublish the graph before editing."
+                    ),
+                }
+                raise Exception(ret)
```

Comment:
Maybe something slightly more specific like `ValidationError`? That way you can be sure all other exceptions get transformed to 500 by the global error hander.